deploy/helms/legion/templates/metrics/graphite.yaml

```diff
@@ -67,6 +67,17 @@ spec:
         - containerPort: 8125
           name: statsd
           protocol: UDP
+        livenessProbe:
+          initialDelaySeconds: 100
+          timeoutSeconds: 10
+          httpGet:
+            path: /
+            port: 80
+        livenessProbe:
```

Comment:
Do we really need liveness and readiness probe in this case?

Reply:
this is two different probes for different ports